app/config/security.yml

```diff
@@ -3,17 +3,17 @@
 
 security:
     providers:
-        fos_userbundle:
-            id: fos_user.user_provider.username_email
+        sylius_user_provider:
+            id: sylius.provider.name_or_email_provider
```

Comment:
I think this should be ``sylius.user_provider.name_or_email``. We have double ``provider`` here, but no indication of what is actually provides...